app/controllers/partners/family_requests_controller.rb

```diff
@@ -29,9 +29,8 @@ def create
       family_requests_attributes = children_grouped_by_item_id.map do |item_id, item_requested_children|
         { item_id: item_id, person_count: item_requested_children.size, children: item_requested_children }
       end
-
       create_service = Partners::FamilyRequestCreateService.new(
-        request_type: params[:request_type],
+        request_type: 2,
```

Comment:
Actually, looking at this, the integers are pretty obscure. Can we change this to a string enum? Similar to [https://github.com/rubyforgood/human-essentials/blob/main/app/models/account_request.rb#L32](https://github.com/rubyforgood/human-essentials/blob/main/app/models/account_request.rb#L32) ?